src/__tests__/README.md

```diff
@@ -0,0 +1,3 @@
+# Tests
+
+Please place your tests in this folder.
```

Comment:
:robot: OpenAI

The README file for tests is empty. Please add more information on how to write and run tests.
```

<!-- This is an auto-generated comment by OpenAI -->